src/main/java/com/slimebot/commands/FdmdsCommand.java

```diff
@@ -92,7 +92,7 @@ public void handleFdmdsModal(ModalInteractionEvent event) {
 			embedBuilder.setFooter(event.getMessage().getEmbeds().get(0).getFooter().getText())
 					.addField("Frage:", question, false);
 		} else if (event.getModalId().contains("send")) {
-			embedBuilder.addField("Frage:", "Heute würde ich gerne von euch wissen, " + question.split(" ", 2)[0].toLowerCase() + " " + question.split(" ", 2)[1], false);
```

Comment:
i think i added this because it's weird when there's a random capitalised word in the middle of a sentence just because the user started with a capital letter when submitting the question

why did you remove this?

Reply:
because it breaks when the user sends a message with just one word. In practice this probably doesn't happen but it would throw an exception if some random troll does so. 
I think a better solution would be `question.substring(0, 1).toLowerCase() + question.substring(1)`